README.md

```diff
@@ -3,31 +3,46 @@
 # os_type
 Rust library to detect the operating system type
 
+## Explain  
+`os_type::current_platform()` return a enum.   
+
+impl  `Debug, Display , To_String` for the enum.
```

Comment:
I'm not sure about this one. This is already done in the library, so it doesn't affect the library user, no?